### ISSUE TITLE
PyPortal crash -- EventCountdown example

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/circuitpython#1680]
I am finding that the PyPortal EventCoundown example causes a (HardFault) reboot to safe mode with the current CP Master build.  https://github.com/adafruit/Adafruit_Learning_System_Guides/tree/master/PyPortal_EventCountdown  Warning -- it may also corrupt the FS.  I have had varying results with this -- Sometimes it runs but the background splash disappears. Sometimes it crashes....

Run on:
Adafruit CircuitPython 4.0.0-beta.5-17-gfce63b17c on 2019-03-21; Adafruit PyPortal with samd51j20

This example ran successfully on previous CP versions -- not sure of latest version tested, but I think I tried Beta5

**Comment by dhalbert** [comment]
I have tried this a few times. I am consistently seeing the background disappear, but otherwise it hasn't crashed. Does it crash right away, or should I let it run a while?

**Comment by jerryneedell** [comment]
@dhalbert  With the default settings, it does not crash immediately -- infact, it seems to be running OK today,
However, if I cahange the event date to:
```
14,16c14,16
< EVENT_MONTH = 6
< EVENT_DAY = 28
< EVENT_HOUR = 17
---
> EVENT_MONTH = 4
> EVENT_DAY = 15
> EVENT_HOUR = 9
```
it crashes immediately on first update. That is, I see the background, then after a few seconds, it crashes.

**Comment by jerryneedell** [comment]
very strange -- I  run eventcountdown.py on the PyPortal, it runs OK -- I then copied eventcountdown.py to new_copy.py  -- When I run new_copy.py  it crashes...
a diff of the files does not report any difference.

In all cases, I am running this from the REPL not code.py.

even stranger,  any of the examples (that is different dates) run OK if loaded as code.py.....
My head hurts...

**Comment by deshipu** [comment]
I think it might depend on where the file is in the partition table, how it is split into blocks, and other things like that. On a non-express board you could just copy the ``current.uf2`` file to reproduce the whole filesystem exactly, perhaps we could have some tool to dump the filesystem off the flash chip on the express boards as well?

**Comment by dhalbert** [comment]
The essence of this bug is described in #1695. I don't know about the hard crashes, but that may be yet some other bug, or else it's a manifestation of the #1695 bug. Could close this in favor of #1680.

**Comment by dhalbert** [comment]
I think this is fixed by #1714. Could you reopen if you are still seeing crashes? Thanks.
